doc/nrf/libraries/networking/lwm2m_location_assistance.rst

```diff
@@ -41,6 +41,85 @@ There are three different supported methods of obtaining the location assistance
 * Query of P-GPS predictions - The P-GPS predictions are queried from nRF Cloud and provided back to the device through the LwM2M server.
   The predictions are stored in the device flash and injected to the GNSS module when needed.
 
+API usage
+*********
+
+This section describes API usage in different scenarios.
+
+Cell-based location
+===================
+
+Cell-based location uses only on current cell or current and neighboring cells.
+To get information about the current cell, use the Connectivity Monitor object (ID 4).
+When you enable the Kconfig option :kconfig:option:`CONFIG_LWM2M_CLIENT_UTILS_CONN_MON_OBJ_SUPPORT`, the connectivity monitor is populated with data about the current cell.
+
+To get information about the neighboring cells, use a collection of ECID-Signal Measurement Information objects (ID 10256).
+To populate the objects, call the :c:func:`lwm2m_ncell_handler_register` function to register the listener for the neighborhood measurements and :c:func:`lwm2m_ncell_schedule_measurement` to schedule a measurement.
+
+The Ground Fix Location object needs to address the ``report_back`` resource before sending a location request.
+Back reporting tells the server whether it needs to send the acquired location back to the device.
+If the location is sent back to the device, the location is stored only in the Ground Fix Location object.
+
+To send the location request for the cell-based location, call the :c:func:`location_assistance_ground_fix_request_send` function.
+
+A-GPS location
```

Comment:
A-GPS doesn't itself provide location, so maybe we should talk about "A-GPS assistance" instead. If you change this, update also the first sentence below.